src/gluonts/torch/model/deepar/module.py

```diff
@@ -151,7 +152,7 @@ def unroll_lagged_rnn(
         output, new_state = self.lagged_rnn(prior_input, input, features)
 
         params = self.param_proj(output)
-        return params, scale, static_feat, new_state
+        return params, scale, output, static_feat, new_state
```

Comment:
I think this change suggests that this method is trying to do too much. But we can re-think this after this PR.